react/InfosCarrousel/styles.styl

```diff
@@ -14,24 +14,26 @@
 
 .InfosCarrousel-navigation
     position absolute
-    bottom rem(10)
-    right spacing_values.xs
+    bottom (spacing_values.xs)
```

Comment:
The `rem(10)` was intended, but perhaps it needs a comment or a variable name - it's there to be visually aligned with buttons inside the `Infos` component.

Reply:
Seen in Zoom ;)